<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_add_Literals</class>
 <widget class="QDialog" name="Dialog_add_Literals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите литерал:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_literal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
